src/Faker/Extension/NumberExtension.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Faker\Extension;
+
+/**
+ * @experimental This interface is experimental and does not fall under our BC promise
+ */
+interface NumberExtension extends Extension
+{
+    /**
+     * Returns a random number between $int1 and $int2 (any order)
+     *
+     * @param int $int1 default to 0
+     * @param int $int2 defaults to 32 bit max integer, ie 2147483647
+     *
+     * @example 79907610
+     */
+    public function numberBetween($int1, $int2): int;
```

Comment:
```suggestion
    public function numberBetween(int $int1, int $int2): int;
```

Reply:
Same here is prev. Dont think int2 is a good name.

Min max maybe?